Items/04_00_WeaponAttachments.txt

```diff
@@ -514,6 +514,21 @@ are excluded.
 == ED Electric Discharge Magazine  $2x + 20 magazine cost ==
     1/2 the bullet's base damage is converted to electrical damage.
 
+== Overcharge Laser Capacitors $(1.5 x mag cost) ==
+
+    Most common laser weapons have after-market ammunition batteries available
+that exceed the recommended voltage for compatibility with that gun. The result
+is a weapon system that imparts more damage through a higher intensity laser, at
+the cost of running the risk of overheating the emitter manifold and causing the
+weapon to "jam." Or in the worst case scenario, to overload even those safety
+measures and to cause the weapon slag, rendering the weapon inoperable and 
+potentially physically burning the user. This attachment is consumed upon use,
+and is only compatible with laser weapons.
+
+Cost: $1.5 x normal mag
+Damage: +15

```

Comment:
Flat damage disproportionate affects miniguns (extreme case) and other fast-firing low-damage weapons. Intentional, or should we make it a % increase?